goss-testing/scripts/check_clock_skew.sh

```diff
@@ -80,6 +80,12 @@ for try in {1..3}; do
       exit_code=1
     fi
   done
-  [[ $exit_code -eq 0 ]] && exit 0
+  [[ $exit_code -eq 0 ]]
```

Comment:
Still need this line?  🤔

Reply:
It might be more straightforward if I just remove the retry logic.  We only had it in there for the slow connections

Reply:
Yes, that seems to work.  I updated the PR to do that.

Reply:
Oh, sorry, I missed this discussion when I was doing my review. I don't see why it would be a problem to retain the retry logic and also include this change. Are we sure the retries will not be needed with this change in place? 

I'm going to try running this in a loop on some systems and see if it ever fails. If it doesn't, then I agree the retry loop is just adding needless complexity.

Reply:
Ok, I've run it in a loop on a few systems for a while now and not seen any failures, so I do agree that the loop appears unneeded. I'll update my review to indicate that just the comment should be changed.